### DISCUSSION TITLE
[QST] Is GPU Direct supported in latest version of RAPIDS?

### DISCUSSION THREAD

**Discussion by Niharikadutta** [NVIDIA/spark-rapids#5345] [General] [answered]
I was wondering if you could shed some light on the support and plans for GPU Direct in Spark RAPIDS? Also I wanted to understand if GPU direct can be enabled on any GPU or does it need the latest A100 series.

**Comment by sameerz** [comment, marked as answer]
GPU Direct storage will be supported for spilling to NVMes local to the host in the near term.  The work is targeting the 0.6 release and is tracked in https://github.com/NVIDIA/spark-rapids/issues/1445 .  GPU Direct storage is supported on Pascal architecture and better (Volta, Turing, Ampere).

**Comment by Niharikadutta** [comment]
Thanks @sameerz!
